backend/theme/admin.py

```diff
@@ -88,13 +96,13 @@ def body_font_preview(self, obj):
         return "No font selected"
 
     def logo_preview(self, obj):
-        if obj.logo_url:
-            return mark_safe(f'<img src="{obj.logo_url}" style="max-height: 50px;"/>')
+        if obj.logo_image:
+            return mark_safe(f'<img src="/upload/{obj.logo_image.file}" style="max-height: 50px;"/>')
         return ""
 
     def background_preview(self, obj):
-        if obj.background_url:
-            return mark_safe(f'<div style="background-image: url({obj.background_url}); height: 50px; width: 100px; background-size: cover;"></div>')
+        if obj.background_image:
+            return mark_safe(f'<div style="background-image: url(/upload/{obj.background_image.file}); height: 50px; width: 100px; background-size: cover;"></div>')
```

Comment:
Same here, you probably have to insert something like the `SUBPATH` from `base_settings.py`

Reply:
You are right though, we can override the `MEDIA_ROOT`, so this should be here instead.